parser/proto.y

```diff
@@ -327,269 +334,231 @@ stringLit : _STRING_LIT {
 		$$ = &stringList{$1, $2}
 	}
 
-aggregate : '{' aggFields '}' {
+messageLiteralWithBraces : '{' messageTextFormat '}' {
 		fields, delims := $2.toNodes()
 		$$ = ast.NewMessageLiteralNode($1, fields, delims, $3)
 	}
 	| '{' error '}' {
 	    $$ = nil
 	}
 
-aggFields : aggField {
```

Comment:
This seems like a more substantial change than was necessary. It's hard to compare the old and new, but it looks like some of the error productions have been removed.

Instead of pushing the permutations of separators into `messageLiteralFields`, I think it's simpler to keep them in an `messageLiteralField` production. That means adding a new `messageLiteralFieldEntry`, which means it would have basically the same shape as I had it (where there was `aggFields`, `aggField`, and `aggFieldEntry`).